packages/fxa-admin-server/scripts/audit-tokens.spec.ts

```diff
@@ -4,20 +4,40 @@
 
 'use strict';
 
-const { assert } = require('chai');
-const util = require('node:util');
-const path = require('path');
-const {
-  auditRowCounts,
-  auditAge,
-  auditOrphanedRows,
-} = require('../../scripts/audit-tokens');
-const mocks = require(`../../test/mocks`);
-const config = require('../../config').getProperties();
-const log = mocks.mockLog();
-const { Account } = require('fxa-shared/db/models/auth/account');
-const { clearDb, scaffoldDb, connectToDb } = require('./db-helpers');
-
+import { assert } from 'chai';
+import util from 'node:util';
+import path from 'path';
```

Comment:
micro nit: love the `node:` prefix on the "node:util" import on L8, why no "node:path" here?
but the tiniest of nits since I don't think we consistently use `node:` for internal packages (but I think I did find a way to enforce that a while ago while poking at some ESLint importer rules and forbidden modules config).

Reply:
I wish I could take credit for the `node:util`, but I think vscode automatically generated these imports. Good call out on consistency issue though.